rhc-ose-ansible/roles/secure-registry/test/secure-registry.yaml

```diff
@@ -0,0 +1,12 @@
+---
+
+- hosts: masters:nodes
+  roles:
+    - role: secure-registry
+      openshift_registry_project: "default"
+      registry_dc: "docker-registry"
+#      registry_certificate: ""
+#      registry_key: ""
+      registry_serviceaccount: "registry"
+      registry_secret_name: "registry-secret"
+      secure_registry_master_host: master1.example.com
```

Comment:
I think we should remove this in the test and allow the tester to define the hosts in inventory.